python/tests/test_lowlevel.py

```diff
@@ -1691,6 +1691,119 @@ def test_divergence_matrix(self):
         with pytest.raises(_tskit.LibraryError, match="UNSUPPORTED_STAT_MODE"):
             ts.divergence_matrix([0, 1], sizes, ids, mode="node")
 
+    def test_pair_coalescence_counts(self):
```

Comment:
We want small tests that do one thing here. (Older tests in tskit were quite sprawling - we're trying to move to better testing).

I would make a new test class here, like
```
class TestPairCoalescecnceErrors:
    def run(sample_sets=None, sample_set_sizes=None, indexes=None, time_windows=None)
         ts = # some reasonable example
         # generate some reansonable defaults for unspecified parameters
        return ts.pair_coalescence_counts(...)
   
@pytest.mark.parametrize("node", [-1, -2, 10**6)
   def test_bad_node_in_sample_sets(self, node):
         with pytest.raises(...)
               self.run(sample_sets=[node])
```

It's a bit tedious to set up, but it's way more useful in terms of finding problems and fixing them.